### DISCUSSION TITLE
Question: hide non-existent routeInfo.json error

### DISCUSSION THREAD

**Discussion by warloque** [react-static/react-static#1442] [General]
Is there a way to handle silently GET errors for routeInfo files for erroneous routes?

For example, URL `app.name/some/wrong/page` produces error **404 (Not Found)** for `app.name/__react-static__/routeInfo/some/wrong/page`

**Comment by SleeplessByte** [comment]
Do you mean in general or during development?

We don't keep a list of all the pages that exist -- that would greatly increase each page's content. So we just try to get the route info for pages. That will fail as a 404 in your log if they don't exist. Nothing we can do about _that_ specifically, but we can "hide" errors passed to the console. Is that what you're referring to?

**Reply by warloque** [reply]
Hiding that 404 for unexsiting reouteInfo's would be an option, of course. 
How can it be done?
